services/housetables/src/main/java/com/linkedin/openhouse/housetables/services/UserTablesServiceImpl.java

```diff
@@ -37,6 +38,10 @@ public class UserTablesServiceImpl implements UserTablesService {
 
   @Autowired UserTablesMapper userTablesMapper;
 
+  // Max length for tableId is 128, minus 22 characters for the suffix "_deleted_" and current
```

Comment:
Some table name can be long. We can have `_del_` prefix and timestamp in sec to shorten the deleted table prefix.

Reply:
timestamp would prefer in millis since that is more precise. Ultimately if the name is long we have to have some sort of conversion that is transparent to users, changing the prefix/suffix only affects the number of tables that need to go through the conversion.